.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+# What does this PR do?
+
+<!--
+Congratulations! You've made it this far! You're not quite done yet though.
+
+Once merged, your PR is going to appear in the release notes with the title you set, so make sure it's a great title that fully reflects the extent of your awesome contribution.
+
+Then, please replace this with a description of the change and which issue is fixed (if applicable). Please also include relevant motivation and context. List any dependencies (if any) that are required for this change.
+
+Once you're done, someone will review your PR shortly (see the section "Who can review?" below to tag some potential reviewers). They may suggest changes to make the code even better. If no one reviewed your PR after a week has passed, don't hesitate to post a new comment @-mentioning the same persons --sometimes notifications get lost.
+-->
+
+<!-- Remove if not applicable -->
+
+Fixes # (issue)
+
+
+## Before submitting
+- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
+- [ ] Did you read the [contributor guideline](https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md),
+      Pull Request section?
+- [ ] Was this discussed/approved via a Github issue? Please add a link
+      to it if that's the case.
+- [ ] Did you make sure to update the documentation with your changes?
+- [ ] Did you write any new necessary tests?
+
+
+## Who can review?
+
+Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
+members/contributors who may be interested in your PR.
+
+<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @
+
+ If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
+ Please tag fewer than 3 people.
+
+Models:
+
+- 
+- 
+
+Layers:
+
+- 
+- 
+
+Losses:
+
+- 
+- 
+-->
+
+<!--
```

Comment:
will this render in our new PRs or are comments excluded?

Reply:
ah, I assume it doesn't render.  Thats great, thanks!

Reply:
This will render as comments.

Reply:
gotcha, if so for now we can probably just remove lines 33 downward because we only have myself and Zhenyu really reviewing at the moment.  If we continue adding more reviewers this makes sense though, thanks!

Reply:
I don't mind being tagged for reviewing model related PRs. But I understand if there are restrictions to that.

Reply:
IMHO It will scale better if we start to use the Github codeowners file:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
Let's  discuss in an Issue outside of this PR - this is a longer conversation and may end up blocking this PR for quite awhile.  better to track in its own issue

Reply:
Probably when the project have merged a batch of their contributions at some point they could be ready to let scale the review system.

P.s. I've not started the thread just a contribution on a started one. Please open a new ticket or better a discussion if you are still interested.